core/templates/pages/topic-editor-page/services/topic-editor-state.service.spec.ts

```diff
@@ -1,763 +0,0 @@
-// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Can you migrate the spec as well instead of deleting it.

Reply:
done